cryptol-remote-api/python/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Revision history for `cryptol` Python package
 
-## 2.12.2 -- YYYY-MM-DD
+## 2.12.2 -- 2021-11-19
```

Comment:
I wouldn't bother putting a date here just yet, as we won't know when 2.12.2's release date will be until it actually happens.